File: NetCore/TaskDto.cs
using System;

namespace NetCore 
{
    class TaskDto 
    {
        public String Id { get; set; }
        public DateTime CreateDate { get; set; }
        public String Name { get; set; }
        public bool IsFinished { get; set; }
        public int Subtasks { get; set; }
        public double Weight { get; set; }
        public String[] Children { get; set; }
        public String Description { get; set; }
        public DateTime Deadline { get; set; }
        public String[] Tags { get; set; }
    }
}

File: Codable/Sources/Benchmark/TaskDto.swift
import Foundation

class TaskClassDto: Codable {

    public var id: String?
    public var createDate: Date?
    public var name: String
    public var isFinished: Bool
    public var subtasks: Int
    public var weight: Double
    public var children: [String]?
    public var description: String?
    public var deadline: Date?
    public var tags: [String]?

    init(
        id: String,
        createDate: Date,
        name: String,
        isFinished: Bool,
        subtasks: Int,
        weight: Double,
        children: [String],
        description: String?,
        deadline: Date?,
        tags: [String]?
    ) {
        self.id = id
        self.createDate = createDate
        self.name = name
        self.isFinished = isFinished
        self.subtasks = subtasks
        self.weight = weight
        self.children = children
        self.description = description
        self.deadline = deadline
        self.tags = tags
    }
}
